symplyphysics/laws/waves/frequency_shift_from_velocity_and_angle.py

```diff
@@ -13,13 +18,20 @@
 ## v is wave velocity in this media,
 ## vo is observer speed relative to the medium (magnitude of the velocity vector),
 ## vs is source speed relative to the medium (magnitude of the velocity vector),
-## pho is angle between vector pointing from source of the wave to the observer (signal vector), and source velocity vector,
-## phs is angle between signal vector and observer velocity vector.
+## pho is angle between vector pointing from source of the wave to the observer (signal vector) and observer velocity vector,
+## phs is angle between signal vector and source velocity vector.
 
 # Conditions:
 ## - Source and observer velocities are less or equal than wave velocity. Otherwise emitted waves are left behind the source or never
 ## reach the observer.
-## - Motion is in 1-D space.
+## - Motion is in 2-D space.
+## - Source speed is constant during one period of the wave.
+
+# NOTE: Signal vector connects the source at the moment of wave emission and observer at the moment
+# of wave reception. It may be quite hard to estimate the point where the observer will be at the
+# moment of observation. Therefore this law is usually applied when objects are moving slowly, or when
+# one of the objects is idle.
+
```

Comment:
In which coordinate system velocities are measured?

Reply:
Updated the description.